mathics/builtin/numbers/algebra.py

```diff
@@ -1884,6 +1915,7 @@ class Collect(_CoefficientHandler):
      = x h[4 Sin[x z]] + x ^ 3 h[1] + x y h[3] + x ^ 2 y h[3] + y ^ 2 h[4 Sin[x z]] + x y ^ 2 h[3] + y ^ 3 h[1]
     """
 
+    summary_text = "collect terms with a variable at the same power"
```

Comment:
Note that in contrast to many of the above, "tests" vs "test" "expands" vs "expand" this uses imperative as seems to be done on the WL site. "list" above is also the imperative tense.

Reply:
List is also a noun. Still, I liked more the present tense.

Reply:
You are correct. In that context "list of ..." it is a noun.